index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="toolbar">
+      <a href="#">
+      <img src="./images/my-logo.svg" alt="my logo" class="my-logo">
+      </a>
+      <nav>
+         <a href="#">
+            <img 
+            src="./images/hamburger.svg" 
+            alt="hamburger menu"
+            class="hamburger-img" id="hamburger">
+            </a>
+      </nav>
+      </header>
+         <section class="main">
+            <header>
+               <h1 class="headline">I'm Ezinwa charles <br>Glad to see you!</h1>
+               <p class="desc">
+                  I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+               </p>
+               <p class="connect-msg">LET'S CONNECT</p>
+            </header>
+              <ul class="social-icons">
+                 <li><a href="#">
+                  <img src="./images/twitter.svg" alt="twitter handle">
+                  </a></li>
+                  <li><a href="#">
+                     <img src="./images/linkedin.svg" alt="linkedin handle">
+                     </a></li>
+                     <li><a href="#">
+                        <img src="./images/media.svg" alt="another social handle">
+                        </a></li>
+                        <li><a href="#">
+                           <img src="./images/github.svg" alt="github handle">
+                           </a></li>
+                        <li>  <a href="#">
+                         <img src="./images/social.svg" alt="social handle">
+                             </a></li>
+              </ul>
```

Comment:
Please sign up for each social media and use your profile link instead of `#` so that people can contact you via your portfolio. kindly fix it.